chocolate-pizza-react/src/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './App.css';
+import Header from './Header.js';
+import Description from './Description.js';
+import Recipe from './Recipe.js';
+import Author from './Author.js';
+import Footer from './Footer.js';
+
+function App() {
+  return (
+    <div className="App">
+      <Header logo={"logo.png"} fb={"fb-icon.png"} twit={"twit-icon.png"} gp={"gp-icon.png"} insta={"insta-icon.png"} flic={"flic-icon.png"} pint={"pint-icon.png"} rss={"rss-icon.png"} mail={"mail-icon.png"}/>
```

Comment:
You can make this more readable by placing each prop in its own line, also, another way to cut down on crowded props is to pass in an array instead.